force-app/main/default/lwc/edaSettings/edaSettings.html

```diff
@@ -0,0 +1,12 @@
+<template>
+    <div class="slds-grid slds-template__container slds-theme_default"  >
+        <div class="slds-template__container slds-col slds-size_4-of-12" >
+            <div class="slds-scrollable_y" style="height:100%;width:30rem">
+                {LabelReference.settingsNavigation}
+            </div>		
+        </div>
+        <div class="slds-col slds-size_8-of-12">
+            {LabelReference.settingsPage}
```

Comment:
Let's use Camel Case here
```suggestion
            {labelReference.settingsPage}
```